modules/wsl-distro.nix

```diff
@@ -50,10 +50,16 @@ with builtins; with lib;
         };
       };
 
-      # WSL is closer to a container than anything else
-      boot.isContainer = true;
+      # We don't need a boot loader
+      boot.loader.grub.enable = false;
+      system.build.installBootLoader = pkgs.writeShellScript "fake-bootloader" "";
+      boot.initrd.enable = false;
+      system.build.initialRamdisk = pkgs.runCommand "fake-initrd" { } ''
+        mkdir $out
+        touch $out/${config.system.boot.loader.initrdFile}
+      '';
+      system.build.initialRamdiskSecretAppender = pkgs.writeShellScriptBin "append-initrd-secrets" "";
```

Comment:
Maybe we can use true here, too?

Reply:
We can't, because the binary needs to be named `append-initrd-secrets`. We could in theory put a symlink to `true` there, but I don't think that would make any difference while being a longer nix expression